easybuild/easyconfigs/b/Boost/Boost-1.67.0-CrayGNU-18.07-python3.eb

```diff
@@ -24,15 +24,15 @@ dependencies = [
 
 patches = ['Boost-1.67.0_python3.patch']
 
-postinstallcmds = ['module load craypkg-gen && craypkg-gen -p %(installdir)s']
+#postinstallcmds = ['module load craypkg-gen && craypkg-gen -p %(installdir)s']
```

Comment:
Can you please explain as comment that this is temporarily commented out until we find a solution for properly using craypkg-gen?